skore/src/skore/project/project.py

```diff
@@ -102,8 +103,18 @@ def __init__(
 
         self._server_info = ServerInfo.rejoin(self)
 
-    def clear(self):
-        """Clear the project."""
+    def clear(self, delete_project=False):
+        """Remove all items from the project.
```

Comment:
we could add a big warning directive to document the behaviour


```suggestion
        """Remove all items from the project.

        .. warning::
           `delete_project=True` will invalidate the `Project` instance. One needs to call
           :class:`~skore.Project` constructor or :func:`skore.open` function to recreate a
           project.
```

Reply:
done